chans/sqlc/mysql/main.go

```diff
@@ -0,0 +1,12 @@
+package main
+
+import (
+	_ "github.com/go-sql-driver/mysql"
+)
+
+func init() {
+	// sql.Register("mysql", &mysql.MySQLDriver{})
```

Comment:
Line commented out.  Does use mean uncommenting and building?

Reply:
I added some comments to explain the above.